### ISSUE TITLE
🐛 Bug Report: can’t create oauth sessions on native apps

### ISSUE THREAD

**Issue by netapy** [appwrite/appwrite#4714]
### 👟 Reproduction steps

When I try to open an oauth session by calling  `await account.createAuth2Session (provider: provider);` on my native mobile app (both android and iOS), I see a white screen and the url is : 
`https://example.com/v1/auth/oauth2/success?project=[PROJECT
ID]&domain=[DOMAIN]&key=[KEY]&secret=[SECRET]#` 

### 👍 Expected behavior

It should close the window and open the session in my app.

### 👎 Actual Behavior

The browser window is white and the session isn’t opened on my app.

### 🎲 Appwrite version

Version 1.1.x

### 💻 Operating system

Linux

### 🧱 Your Environment

My client runs the Flutter SDK.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Thanks for raising this issue! 🙏🏼 I think the problem might be because we implemented the route as

```
https://example.com/auth/oauth/success
```

rather than

```
https://example.com/v1/auth/oauth2/success
```

**Comment by netapy** [comment]
@stnguyen90  thanks for handling this.

Can't we add this to the 1.1.1 ? I would really need this as fast as possible and unfortunately I can't downgrade to 1.0.3 because it causes some database issues (unless you know how to solve these?).
Cheers

**Comment by stnguyen90** [comment]
@netapy we can't add it to 1.1.1 because 1.1.1 was already released 😕

**Comment by netapy** [comment]
@stnguyen90 Oh right !
I saw your pull request, any idea how long it'll take to implement the fix ? Sorry for insisting but it's really important for me.

**Comment by stnguyen90** [comment]
@netapy, hopefully in the next couple of days.

**Comment by Shiba-Kar** [comment]
Facing the same issue in Flutter

**Comment by sampathkumar7** [comment]
I'm facing same issue

**Comment by sampathkumar7** [comment]
anyway i can change that url manually, i'm using digitalocean onclick droplet

**Comment by netapy** [comment]
Hi, I believe the PR for this issue has been accepted (https://github.com/appwrite/appwrite/pull/4725).
Has it been pushed to the latest docker tag yet ? This seems to be the only issue still open for 1.1.2.

**Comment by stnguyen90** [comment]
@netapy, yes, this should be resolved in 1.1.2.
